modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -72,6 +72,11 @@ A batch job contains one or more batch steps that act upon records as they move
 Each batch step in a batch job contains processors that act upon a record to transform, route, enrich, or otherwise process data contained within it. By leveraging the functionality of existing Mule processors, the batch step offers a lot of flexibility regarding how a batch job processes records. See xref:batch-filters-and-batch-aggregator.adoc[Refining Batch Steps Processing] for more information.
 
 A batch job executes when the flow reaches the process-records section of the batch job. When triggered, Mule creates a new batch job instance.
+When the job instance becomes executable, the batch engine submits a task for each record block to the
+I/O pool to process each record. Parallelism occurs automatically, at the record
+block level. To control the number or threads to use and the rate at which these processing tasks execute,
+the Mule runtime engine automatically makes tuning decisions based on a set of processing types that are also configurable
```

Comment:
`To control the number or threads to use and the rate at which these processing tasks execute,
the Mule runtime engine automatically makes tuning decisions based on a set of processing types that are also configurable` -> `The Mule Runtime uses its auto-tuning capabilities to determine how many threads are to be used and the level of parallelism to be applied`

Reply:
done